frontend/src/components/filters.svelte

```diff
@@ -87,8 +86,8 @@
           <input
             type="checkbox"
             class="checkbox"
-            bind:checked={movieChecked}
-            disabled={!tvChecked}
+            bind:checked={$filters.movieChecked}
+            disabled={!$filters.tvChecked}
             on:change={() => search()}
           />
         </label>
```

Comment:
We are mixing checkboxes and toggles in the filter menu, it should probably just all be toggled for that modern look.

Also, both of the checkboxes in `Content type` goes gray when one is deselected - It should only be the one you shouldn't press.